### PR TITLE
Excluding tests on windows 32bit

### PR DESCRIPTION
**Pull request by adamfarley**
Some tests that are already excluded on windows 64bit should also
be excluded on windows 32bit for the same reasons.

I've also tidied up some formatting, as tabs seem the rule for this
file, and it seems a few spaces snuck in. Fixed.

Signed-off-by: Adam Farley <adfarley@redhat.com>

### CODE REVIEW

**Review by sxa — COMMENTED** [review]
I think I'm ok with this, although all the formatting changes make it harder to review than I'd like. It looks like there's just two extra sections that have been added here for win32 - is that correct?
I'll defer to Shelley for formal final approval though :-)

**Comment by adamfarley** [comment]
> 
> 
> I think I'm ok with this, although all the formatting changes make it harder to review than I'd like. It looks like there's just two extra sections that have been added here for win32 - is that correct?
> I'll defer to Shelley for formal final approval though :-)

Yes, that's correct. Apologies for making things harder for reviewers with the formatting.

For other reviewers, most of the changes are formatting changes. The only real additions are on lines 465-469 and 1150-1154.

**Review by sxa — APPROVED** [review]
Thanks for the clarification Adam!

This repo requires two approvals, so I'm happy to provide one of them ;-)

**Comment by gdams** [comment]
Reviewing with whitespace changes hidden make things much easier here https://github.com/adoptium/aqa-tests/pull/2827/files?diff=unified&w=1

**Review by gdams — APPROVED** [review]

**Review by smlambert — APPROVED** [review]

**Pull request merged**
